apps/docs/src/app/platform/component-docs/platform-forms/multi-input/platform-multi-input-docs.module.ts

```diff
@@ -0,0 +1,61 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule, Routes } from '@angular/router';
```

Comment:
Api file is not generated. please check.

Reply:
not sure which file ur talking abt, pls clarify more details.

Reply:
https://deploy-preview-3649--fundamental-ngx.netlify.app/#/platform/multi-input/api
Api's are not displayed here.